count1_test.go

```diff
@@ -19,15 +19,15 @@ var (
 )
 
 func BenchmarkCountBadIPs1(b *testing.B) {
-	badIPs, flows, _ := getFlowsAndBadIPs(benchNumFlows, benchNumBadIPs, benchHitRate)
+	badIPs, flows, _ := getBadIPsAndFlows(benchNumFlows, benchNumBadIPs, benchHitRate)
```

Comment:
Argument order is wrong, it should be:
```suggestion
	badIPs, flows, _ := getBadIPsAndFlows(benchNumBadIPs, benchNumFlows, benchHitRate)
```

Reply:
Thanks! I've just removed the arguments as they're not actually used (as you pointed out).